internal/glfw/generate.sh

```diff
@@ -0,0 +1,16 @@
+VERSION=3.3.8
+
+curl -L https://github.com/glfw/glfw/releases/download/$VERSION/glfw-$VERSION.zip > glfw-$VERSION.zip
+unzip glfw-$VERSION.zip
+cd glfw-$VERSION/
+mkdir build
+cd build
+export MACOSX_DEPLOYMENT_TARGET=10.14
+cmake -D CMAKE_BUILD_TYPE=Release -D GLFW_NATIVE_API=1 -D CMAKE_OSX_ARCHITECTURES="x86_64;arm64" -D BUILD_SHARED_LIBS=ON -D CMAKE_C_COMPILER=clang ../
```

Comment:
Can we make our own build process?

We used to have our own build process for Windows: https://github.com/hajimehoshi/ebiten/blob/2.3/internal/glfw/gen.go

Some options like `-Wl,--no-insert-timestamp` is important in order to make a deterministic build by the way.

Reply:
I can look into it. It might take some time to figure it out.

Reply:
Sure, thank you for working on this!

Reply:
Did you store the GLFW source in the tree? Or should I download it when the generator runs?

Reply:
https://github.com/hajimehoshi/ebiten/tree/2.3/internal/glfw/glfw

I stored this for Windows because I added some modification to the source.

For macOS, I think getting the source code via `go list -f {{.Dir}} github.com/go-gl/glfw/v3.3/glfw` would work.